deployments/tekton.go

```diff
@@ -360,18 +361,29 @@ func applyTektonStaging(c *kubernetes.Cluster, ui *termui.UI) (string, error) {
 }
 
 func (k Tekton) createGiteaCredsSecret(c *kubernetes.Cluster) error {
-	_, err := c.Kubectl.CoreV1().Secrets(TektonStagingNamespace).Create(context.Background(),
+	// See internal/cli/clients/gitea/gitea.go, func
+	// `getGiteaCredentials` for where the cli retrieves the
+	// information for its own gitea client.
+	//
+	// See deployments/gitea.go func `apply` where `install`
+	// configures gitea for the same credentials.
+
+	giteaAuth, err := GiteaInstallAuth()
```

Comment:
The memoization of this method will be a problem if we ever want to support resuming an incomplete installation but let's solve this problem when (and if) we get there.

Reply:
Agreed. My handling of a partial install is to `uninstall` and then start fresh.